### ISSUE TITLE
Tab order and/or set should be changeable while running

### ISSUE THREAD

**Issue by grmcdorman** [grmcdorman/esp8266_web_settings#10]
Currently, the web settings can only be added, and the first tab is always the default tab.

A desirable enhancement would be to change the set on the fly. This could be replacing the entire list - e.g. by providing a `clear` operation - or by some form of "reorder" operation.

In particular, it may be appropriate to have different tabs in Soft AP mode v.s. when connected to an AP; in Soft AP mode it would be nice to have the WiFi settings be the first (i.e. default) tab, instead of an overview panel.
